src/cartservice/src/appsettings.json

```diff
@@ -11,5 +11,6 @@
       "EndpointDefaults": {
         "Protocols": "Http2"
       }
-    }
+    },
+    "Urls": "http://*:7070"
```

Comment:
Where is this used?  is it required?

Reply:
This is not required and was simply added when I was running the application natively and needed to replicate the port set within the Dockerfile here: https://github.com/GoogleCloudPlatform/microservices-demo/blob/a7ad8f7cc7cd15e7db024c7b554b04e0d1fbbe52/src/cartservice/Dockerfile#L32

I will revert this change.